error-prone-contrib/src/test/resources/tech/picnic/errorprone/refasterrules/MicrometerRulesTestOutput.java

```diff
@@ -0,0 +1,60 @@
+package tech.picnic.errorprone.refasterrules;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSet;
+import io.micrometer.core.instrument.Tag;
+import io.micrometer.core.instrument.Tags;
+import tech.picnic.errorprone.refaster.test.RefasterRuleCollectionTestCase;
+
+final class MicrometerRulesTest implements RefasterRuleCollectionTestCase {
+  @Override
+  public ImmutableSet<Object> elidedTypesAndStaticImports() {
+    return ImmutableSet.of(ImmutableList.class);
+  }
+
+  ImmutableSet<Iterable<Tag>> testTagsOf1() {
+    return ImmutableSet.of(Tags.of(Tag.of("foo", "bar")), Tags.of(Tag.of("foo", "bar")));
```

Comment:
We try to use distinct values in a test such that we are always sure it replaced the correct thing. Because the TagsOf4 and TagsOf5 have too many values to only use words like foo bar baz qux etc. I added the v1 v2 as values.